module2/src/com/epam/arraysofarrays/MatrixWorkerOrderN.java

```diff
@@ -0,0 +1,69 @@
+package com.epam.arraysofarrays;
+
+public class MatrixWorkerOrderN {
```

Comment:
It is better to rename e.g. 
```suggestion
public class MatrixBuilder {
```